lib/TypeNames/LLVMTypeNames.cpp

```diff
@@ -61,175 +61,132 @@ bool isScalarCType(const llvm::Type *LLVMType) {
   return false;
 }
 
-std::string getScalarCType(const llvm::Type *LLVMType, const CBuilder &B) {
```

Comment:
Nothing this here but it applies to a bunch of other places in this file.

Why are we losing `const` on the builder? Is it really necessary?

Reply:
It's a bit more nuanced.

We lost this `const`ness a while back when the first iteration of `NameBuilder` got merged (because it was stateful, and `mutable` is evil).

The PR that got merged the other day finally restored that `const`ness. I fixed all the cases I ran into, but it looks like I missed some. Thanks for pointing this out!

Reply:
Done.